env0/resource_template.go

```diff
@@ -239,6 +239,12 @@ func getTemplateSchema(prefix string) map[string]*schema.Schema {
 			ConflictsWith: allVCSAttributesBut("helm_chart_name", "is_helm_repository"),
 			RequiredWith:  requiredWith("helm_chart_name"),
 		},
+		"terragrunt_tf_binary": {
```

Comment:
Could not set a deault value to "OpenTofu".
Default will only make sense if this was a "Terragrunt" template.
This is also an issue with existing templates that will use "Terrafrom" and not "OpenTofu".

I tried a few options. At the end I had to manually handle this unique use-case.
This makes the PR a little more complex than usual.